Dockerfile.debug

```diff
@@ -3,6 +3,12 @@ FROM golang:1.16-buster AS builder
 # Copy sources
 WORKDIR $GOPATH/src/github.com/oauth2-proxy/oauth2-proxy
 
+
+ARG GITHUB_TOKEN
+RUN if [ -z "$GITHUB_TOKEN" ]; then echo "missing GITHUB_TOKEN env var which is needed for asset downloads"; false; fi
+# update git config to use our token for pulling assets
+RUN printf "[url \"https://%s@github.com/\"]\n\tinsteadOf = https://github.com/\n" ${GITHUB_TOKEN} >> /root/.gitconfig
+
```

Comment:
We prob need to add `ARG GOPRIVATE="github.com/dynata/*,github.com/researchnow/*"` so go mod download will handle priv repository.

Reply:
Added